clouddriver-web/config/clouddriver.yml

```diff
@@ -64,7 +65,7 @@ azure:
   defaults:
 
   accounts:
-    - name: default
+    - name: azure-cred1
```

Comment:
this will not go into the netflix PR but since it is checked in as part of my branch changes, it does show in this PR.